forge-kernel/src/ft/binding_ft.cpp

```
@@ -0,0 +1,111 @@
+#include <napi.h>
+
+#include <string>
+
+#include "forge/ft/FeatureTree.hpp"
+
+namespace forge {
+namespace bind {
+namespace {
+
+template <typename Fn>
+Napi::Value safe(const Napi::CallbackInfo& info, Fn&& fn) {
```

Clang (14.0.6)

[warning] use a trailing return type for this function

(modernize-use-trailing-return-type)